example/src/main/java/com/stripe/example/controller/AsyncTaskTokenController.java

```diff
@@ -17,6 +18,7 @@
 public class AsyncTaskTokenController {
 
     private CardInputWidget mCardInputWidget;
+    private Context mContext;
```

Comment:
making the example project work with the new required Context arg. This is an example of how this is a breaking change.